openpype/hosts/nuke/api/plugin.py

```diff
@@ -818,17 +817,11 @@ def __init__(self,
 
         self.file = self.fhead + self.name + ".{}".format(self.ext)
```

Comment:
Let's match the code style here with what we're doing in the if statement.
```suggestion
        self.file = "{}{}.{}".format(self.fhead, self.name, self.ext)
```

Reply:
Also, what's the reason we're not doing `if ... else`?

```python
        if ".{}".format(self.ext) in VIDEO_EXTENSIONS:
            # Output is video
            self.file = "{}{}.{}".format(self.fhead, self.name, self.ext)
        else:
            # Output is expected to be image (or image sequence)
            # When the file is an image it's possible it
            # has extra information after the `fhead` that
            # we want to preserve, e.g. like frame numbers
            # or frames hashes like `####`
            filename_no_ext = os.path.splitext(
                os.path.basename(self.path_in))[0]
            after_head = filename_no_ext[len(self.fhead):]
            self.file = "{}{}.{}.{}".format(
                self.fhead, self.name, after_head, self.ext)
```